api/mesh/v1alpha1/mesh.proto

```diff
@@ -83,6 +83,21 @@ message CertificateAuthorityBackend {
 
   // Configuration of the backend
   google.protobuf.Struct conf = 4;
+
+  enum Mode {
+    // A STRICT mode implies server validates the connection and accepts only
```

Comment:
I think -> `implies that the server`... same below. Put this text through Grammarly.